src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -255,14 +322,37 @@ const CreateRoomForm = ({ onFormSubmit }) => {
                         />
                         <Box m={4} />
                         <Box display="flex" justifyContent="flex-end">
-                            <Button
-                                startIcon={<CheckBoxIcon />}
-                                variant="contained"
-                                color="primary"
-                                disabled={isSubmitting}
-                                onClick={submitForm}>
-                                {t('form.submit')}
-                            </Button>
+                            {isEditing ? (
```

Comment:
to avoid button duplication. Maybe?
```
{isEditing && (
    <Button
        color="primary"
        disabled={isSubmitting}
        onClick={() => {
            onCreateFormReset()
            cancelEdit()
            setValue([])
        }}>
        {t('form.buttons.cancel')}
    </Button>
)}
<Button
    startIcon={<CheckBoxIcon />}
    variant="contained"
    color="primary"
    disabled={isSubmitting}
    onClick={submitForm}>
    {isEditing ? t('form.buttons.save') : t('form.buttons.submit')}
</Button>
```